src/plone/rest/tests/test_traversal.py

```diff
@@ -126,3 +126,7 @@ def test_json_request_to_view_namespace_returns_view(self):
         self.portal[self.portal.invokeFactory('Folder', id='folder1')]
         obj = self.traverse('/plone/folder1/@@folder_contents')
         self.assertTrue(IBrowserView.providedBy(obj), 'IBrowserView expected')
+
+    def test_json_request_to_portal_resource_returns_page_template(self):
```

Comment:
@displacedaussie returning a page template on a JSON request violates the HTTP specs (content negotiation). If portal_resource is actually doing this should be fixed in Zope/Plone, not in plone.rest.

Reply:
I don't believe zope/plone is https://github.com/plone/plone.app.theming/blob/master/src/plone/app/theming/browser/themefile.py#L25.